src/Dialect/ONNX/Rewrite.cpp

```diff
@@ -525,6 +525,7 @@ class RNNOpRewriteLayoutPattern : public OpRewritePattern<ONNXOp> {
           rewriter.getIntegerAttr(
               rewriter.getIntegerType(64, /*isSigned=*/true), 0));
       // Update the output shape.
+      resetTypesShapeToQuestionmarks(onnxOp);
```

Comment:
Could you please put some additional comments here (something similar to the comments for `resetTypesShapeToQuestionmarks`)? Though it may be duplicated, it helps other developers know the reason when scanning the code here?

Reply:
Good suggestion. Added comments at your request.